App Categories.xml

```diff
@@ -1,3 +1,53 @@
 <string-array name="app_categories">
+    <item>Art &amp; Design</item>
```

Comment:
does this map 1-1 with the categories in the Google Play store?

Reply:
Sorry, I didn't get you meant whether the Art and Design category exists or not.

Reply:
@YohanesGetinet1 no I was talking about all the categories in general. Are these the exact same categories that you see for the apps on the Google Play Store?

Reply:
Ok, @vinaygaba thanks for the clarity 
most of the categories are in Google Play Store but some of the categories are sub-categories for the main ones. Should I have to include the main categories only?